app/controllers/challenges_controller.rb

```
@@ -148,6 +149,12 @@ def set_s3_direct_post
                             acl:                   'private')
   end
 
+  def create_invitations
+    params[:challenge][:invitation_email].split(',').each do |email|
+      @challenge.invitations.create(email: email.strip)
```

You should use methods with `!` whenever it makes sense to follow policy of failing fast. Here it would be `create!`.

---

Done!